### ISSUE TITLE
Product Idea: Unstar

### ISSUE THREAD

**Issue by selfagency** [astralapp/astral#141]
Add an unstar button next to edit tags

**Comment by syropian** [comment]
Funny you mention it, I actually used to have this in v1 but it required an extra level of permissions on the OAuth scope that a lot of people weren't overly thrilled about, so I took it out for v2. However I _think_ it's possible to grant certain permissions after the fact by having users opt-in to a different scope later. I'll have to do some research around it.

**Comment by selfagency** [comment]
Yes, I've been using it since it launched and I was like, "Wasn't there an unstar button here...?"

I've also been experiencing a lot of bugginess. I can't change tag names—it throws an error. Sometimes adding tags creates duplicate tags and clicking on the duplicate tag causes the whole app to go buggy. Stars don't refresh when you click the refresh button. Would also be awesome if you could add search operators, so I can type in something like `javascript !jquery` and have it bring back all the results not containing jQuery.

**Comment by mr-shitou** [comment]
is this possible use personal-access-token
https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
plus, i think it's not necessary to access all user date, maybe just public data only can do the tag function.
and other repo do it use gists, maybe a good way
